Core/include/Acts/Geometry/GeometryIdentifier.hpp

```diff
@@ -74,18 +78,20 @@ class GeometryIdentifier {
   constexpr GeometryIdentifier& setSensitive(Value sensitive) {
     return setBits(kSensitiveMask, sensitive);
   }
+  /// Set the extra identifier
+  constexpr GeometryIdentifier& setExtra(Value extra) {
+    return setBits(kExtraMask, extra);
+  }
 
  private:
-  // (2^8)-1 = 255 volumes
-  static constexpr Value kVolumeMask = 0xff00000000000000;
-  // (2^8)-1 = 255 boundaries
-  static constexpr Value kBoundaryMask = 0x00ff000000000000;
-  // (2^12)-1 = 4096 layers
-  static constexpr Value kLayerMask = 0x0000fff000000000;
-  // (2^8)-1 = 255 approach surfaces
-  static constexpr Value kApproachMask = 0x0000000ff0000000;
-  // (2^28)-1 sensitive surfaces
-  static constexpr Value kSensitiveMask = 0x000000000fffffff;
+  // clang-format off
+  static constexpr Value kVolumeMask    = 0xff00000000000000; // (2^8)-1 = 255 volumes
+  static constexpr Value kBoundaryMask  = 0x00ff000000000000; // (2^8)-1 = 255 boundaries
+  static constexpr Value kLayerMask     = 0x0000fff000000000; // (2^12)-1 = 4096 layers
+  static constexpr Value kApproachMask  = 0x0000000ff0000000; // (2^8)-1 = 255 approach surfaces
+  static constexpr Value kSensitiveMask = 0x000000000fffff00; // (2^20)-1 sensitive surfaces
+  static constexpr Value kExtraMask     = 0x00000000000000ff; // (2^8)-1 extra values
```

Comment:
should we add the explicit number of combinations here too?

```suggestion
  static constexpr Value kSensitiveMask = 0x000000000fffff00; // (2^20)-1 = 1048575 sensitive surfaces
  static constexpr Value kExtraMask     = 0x00000000000000ff; // (2^8)-1 = 255 extra values
```

Reply:
Sure, before it was hundreds of millions, so I didn't bother